<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#tabWidget{
background-image: url(:/new/prefix2/resource/restaurant.png);}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>854</x>
      <y>480</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tester connexion</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>721</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Modifier">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QLabel" name="nom_label_Edit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ingredient_LE_Edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_LE_Edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BackToConfig_B">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#BackToConfig_B{
image: url(:/new/prefix2/resource/SUPP.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="ConfrimEdit_B">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#ConfrimEdit_B{

image: url(:/new/prefix2/resource/VALID.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="id_label_Edit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_LE_Edit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="ingredient_label_Edit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>ingredient</string>
      </property>
     </widget>
     <widget class="QLabel" name="categorie_label_Edit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>categorie</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prix_LE_Edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="prix_label_Edit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>200</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="categorie_LE_Edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Affichage</string>
     </attribute>
     <widget class="QTableView" name="plat_tab">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>711</width>
        <height>301</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>541</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Client</string>
      </property>
      <widget class="QLabel" name="id_label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLabel" name="nom_label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="prenom_label">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ingredient</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_LE">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_LE">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ingredient_LE">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter_B">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>210</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
#ajouter_B{
image: url(:/new/prefix2/resource/+.png);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="categorie_LE">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="prenom_label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>categorie</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Prix_LE">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="prenom_label_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>prix</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Config">
     <attribute name="title">
      <string>Config</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Id :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_LEdelete">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>55</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_B">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#delete_B{
image: url(:/new/prefix2/resource/SUPP.png);;}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="idList_CB">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>145</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;</string>
      </property>
      <property name="text">
       <string>Id :</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(0, 0, 0, 0.1);
border-radius: 15px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="ListEdit_B">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#ListEdit_B{
image: url(:/new/prefix2/resource/modi.jfif);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="ListDelete_B">
       <property name="geometry">
        <rect>
         <x>104</x>
         <y>70</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#ListDelete_B{
image: url(:/new/prefix2/resource/SUPP.png);}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <zorder>frame</zorder>
     <zorder>label</zorder>
     <zorder>id_LEdelete</zorder>
     <zorder>delete_B</zorder>
     <zorder>idList_CB</zorder>
     <zorder>label_2</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
